tests/store_test.js

```diff
@@ -15,7 +16,7 @@ Scenario('account registration', ({ I, homePage, registerPage }) => {
     registerPage.verifyRegisterAccountText();
     registerPage.submitRegistrationForm(registrationUser);
 
-    registerPage.verifyRegisterSuccessText();
+    registerPage._verifyRegisterSuccessText();
```

Comment:
underscore means method is for inner usage in object, but you use it outside of object(in scenario)